hello-world/tutorial/index.md

```diff
@@ -0,0 +1,494 @@
+
+## Prerequisites 
+
+This tutorial uses [Starport](https://github.com/tendermint/starport) v0.15.0. The Starport tool is the easiest way to build a blockchain. 
+
+To install `starport` into `/usr/local/bin`, run the following command:
+
+```
+curl https://get.starport.network/starport@v0.15.0! | bash
+```
+
+You can also use Starport v0.15.0 in a [browser-based IDE](http://gitpod.io/#https://github.com/tendermint/starport/tree/v0.15.0). For more installation options, see [Install Starport](https://github.com/tendermint/starport/blob/develop/docs/1%20Introduction/2%20Install.md).
+
+## Create a Blockchain App
+
+Create a blockchain app with a blog module to write posts on other blockchains that contain the Hello World message (or rather Hello Mars, Hello Cosmos and Hello Earth) for an IBC module.
+
+For this simple example, create an app that contains a blog module that has a post transaction with title and text. 
+
+After you define the logic, run two blockchains that have this module installed.
+- The chains can send posts between each other using IBC. 
+- On the sending chain, save the `acknowledged` and `timed out` posts.
+
+After the transaction is acknowledged by the receiving chain, you know that the post is saved on both blockchains.
+- The sending chain has the additional data `postID`. 
+- Sent posts that are acknowledged and timed out contain the title and the target chain of the post. These identifiers are visible on the parameter `chain`. 
+The following chart shows the lifecycle of a packet that travels through IBC.
+
+[![The Lifecycle of an IBC packet in the blog module](./packet_sendpost.png)](./packet_sendpost.png)
+
+## Initialize the Blockchain
+
+Scaffold a new blockchain named `planet`
+
+```go
+starport app github.com/tuto/planet
+cd planet
+```
+
+A new directory named planet is created. This directory contains a working blockchain app.
+
+Scaffold a module inside your blockchain named `blog` with IBC capabilities.
+The blog module contains the logic for creating blog posts and routing them through IBC to the second blockchain.
+
+```go
+starport module create blog --ibc
+```
+
+A new directory with the code for an IBC module is created in `planet/x/blog`. 
+The `--ibc` flag includes all the logic for the scaffolded IBC module.
+
+1. Create the transction types for the module. 
+Use the `starport type` command to scaffold a transaction message with Starport.
+The module has the logic for:
+
+- Creating Blog posts
+- Processing acknowledgments for sent posts
+- Managing post time outs 
+
+```go
+starport type post title content --module blog
+starport type sentPost postID title chain --module blog
+starport type timedoutPost title chain --module blog
+```
+
+These three `starport type` commands create the foundation of the transaction handlers.
+
+To manage multiple modules within your Starport app, use the `--module` flag to define which module the new transaction type is added to.
```

Comment:
The first argument of the `starport type` command specifies the name of the type being created, the following arguments define fields associated with the type. `--module` is an optional flag that allows to specify in which module the type will be scaffolded (without this flag the type is scaffolded in the module that matches the name of the repo).